### PR TITLE
31 recognition and award

### PR DESCRIPTION
**Pull request by Bhargav-44**
Added the recognition and reward page

### CODE REVIEW

File: src/components/About/RecognitionAward/index.astro [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+const survey_pts = [
+    "Institute has been Certified for ISO 9001:2000 in A.Y. 2005-06 and currently upgraded to ISO: 9001:2015 in A.Y.2017-18",
+    "Institute has received Asia Pacific Quality Organization (APQO) World Class Global Performance Excellence Award 2016 on 21 November 2016 in New Zealand for Best in Class Award in Education Category.",
+    "Recipient of IMC Ramakrishna Bajaj National Quality Award (RBNQA) in 2015 for Performance Excellence and Commendation certificate in 2013.",
+    "Best Innovative & Performing Engineering College NEA (National Maharashtra Education Award) in A.Y.2015-16 organized at UOM",
+];
+---
+
+<section class="px-5">
```
Comment (JayeshVP24) [inline src/components/About/RecognitionAward/index.astro:67]: remove px-5
add `p-4 lg:p-16`

File: src/pages/about/recognition-and-award.astro [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (JayeshVP24) [inline src/pages/about/recognition-and-award.astro:1]: rename recognition-and-award -> recognition-and-awards
awards is plural

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]
Your margin tops across all headings are not enough for large screens. increase it for large screens

File: src/components/About/RecognitionAward/index.astro [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+const survey_pts = [
+    "Institute has been Certified for ISO 9001:2000 in A.Y. 2005-06 and currently upgraded to ISO: 9001:2015 in A.Y.2017-18",
+    "Institute has received Asia Pacific Quality Organization (APQO) World Class Global Performance Excellence Award 2016 on 21 November 2016 in New Zealand for Best in Class Award in Education Category.",
+    "Recipient of IMC Ramakrishna Bajaj National Quality Award (RBNQA) in 2015 for Performance Excellence and Commendation certificate in 2013.",
+    "Best Innovative & Performing Engineering College NEA (National Maharashtra Education Award) in A.Y.2015-16 organized at UOM",
+];
+---
+
+<section class="p-4 lg:p-16">
```
Comment (JayeshVP24) [inline src/components/About/RecognitionAward/index.astro:67]: You have made proper divs.
But you added pt pb everywhere and one spacing change requires us to change many lines of code. Tailwind has something called space-y space-x.
Read the tailwind docs. and u can remove all singular padding and use space so spacing is consistent all over the page

File: src/components/About/RecognitionAward/index.astro [hunk 4]
```diff
@@ -0,0 +1,196 @@
+---
```
Comment (JayeshVP24) [inline src/components/About/RecognitionAward/index.astro:1]: Also this whole page is unnecessary. It beats the point of components. Move all this code to pages/about/recognition-and-awards.astro page

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/pages/about/recognition-and-awards.astro [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+    "Institute has been Certified for ISO 9001:2000 in A.Y. 2005-06 and currently upgraded to ISO: 9001:2015 in A.Y.2017-18",
+    "Institute has received Asia Pacific Quality Organization (APQO) World Class Global Performance Excellence Award 2016 on 21 November 2016 in New Zealand for Best in Class Award in Education Category.",
+    "Recipient of IMC Ramakrishna Bajaj National Quality Award (RBNQA) in 2015 for Performance Excellence and Commendation certificate in 2013.",
+    "Best Innovative & Performing Engineering College NEA (National Maharashtra Education Award) in A.Y.2015-16 organized at UOM",
+];
+---
+
+<Layout title="Welcome to TCET Mumbai" description='Thakur College of Engineering and Technology' >
+	<main>
+		<section class="p-4 lg:p-16 space-y-4 lg:space-y-10">
```
Comment (JayeshVP24) [inline src/pages/about/recognition-and-awards.astro:70]: remove section, you coded a whole page not a section.
your sections should be on  the divs

File: src/pages/about/recognition-and-awards.astro [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+				has identified with right from its inception in the academic year
+				2001-02. Today institutes have well-designed and well-executed systems
+				and processes, maintaining transparency, accountability and integrity.
+				Quality of product and services offered to the students (our customer)
+				is second to none in the education sector particularly professional
+				technical education.
+			</p>
+		
+			<div
+				class="h-full w-full flex flex-wrap m-auto lg:flex-nowrap items-start justify-center mt-6"
```
Comment (JayeshVP24) [inline src/pages/about/recognition-and-awards.astro:95]: remove lg:no-wrap

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

File: src/pages/about/recognition-and-awards.astro [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+    "Institute has been Certified for ISO 9001:2000 in A.Y. 2005-06 and currently upgraded to ISO: 9001:2015 in A.Y.2017-18",
+    "Institute has received Asia Pacific Quality Organization (APQO) World Class Global Performance Excellence Award 2016 on 21 November 2016 in New Zealand for Best in Class Award in Education Category.",
+    "Recipient of IMC Ramakrishna Bajaj National Quality Award (RBNQA) in 2015 for Performance Excellence and Commendation certificate in 2013.",
+    "Best Innovative & Performing Engineering College NEA (National Maharashtra Education Award) in A.Y.2015-16 organized at UOM",
+];
+---
+
+<Layout title="Welcome to TCET Mumbai" description='Thakur College of Engineering and Technology' >
+	<main>
+		<div class="p-4 lg:p-16 space-y-4 lg:space-y-10">
```
Comment (JayeshVP24) [inline src/pages/about/recognition-and-awards.astro:70]: remove this div and move the class to main

**Review by JayeshVP24 — CHANGES_REQUESTED** [review]

**Review by kamalika0363 — APPROVED** [review]
Approved

**Pull request merged**